src/components/conjunction/ConjunctionAccordion.tsx

```diff
@@ -12,27 +13,24 @@ import { ConjunctionMissDistanceChart } from './ConjunctionMissDistanceChart';
 import { ConjunctionObjectData } from './ConjunctionObjectData';
 
 type ConjunctionAccordionType = {
-  id: string;
-  haveMtp: boolean;
+  shortId: string;
+  event: TypeEventSummaryOut;
   primaryObject: TypeSatelliteOut;
   secondaryObject: TypeSatelliteOut | null;
-  events: TypeEventSummaryOut[];
-  event: TypeEventSummaryOut;
-  dataSources: TypeDataSourcesOut;
-  handleDownloadData: () => Promise<unknown>;
+  isSpecial: boolean;
 };
 
-const ConjunctionAccordion = ({
-  id,
-  haveMtp,
+const ConjunctionAccordion = async ({
+  shortId,
+  event,
   primaryObject,
   secondaryObject,
-  events,
-  event,
-  dataSources,
-  handleDownloadData,
+  isSpecial,
 }: ConjunctionAccordionType) => {
-  const t = useTranslations('Accordions.Conjunction');
+  const t = await getTranslations('Accordions.Conjunction');
```

Comment:
if accordion is not clinet anymore we can add translations as a nested key for page translations